pom.xml

```diff
@@ -122,7 +122,10 @@
     <gson.version>2.8.9</gson.version>
     <gson-extras.version>0.2.2</gson-extras.version>
     <org-json.version>20240205</org-json.version>
+    <jackson.version>2.12.7.20240502</jackson.version>
```

Comment:
it's the jackson-bom version https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom